pkg/commands/compute/deploy.go

```diff
@@ -44,12 +44,19 @@ type DeployCommand struct {
 	Manifest       manifest.Data
 	Path           string
 	Domain         string
-	Backend        string
-	BackendPort    uint
+	Backend        Backend
 	Comment        cmd.OptionalString
 	ServiceVersion cmd.OptionalServiceVersion
 }
 
+// Backend represents the configuration parameters for a backend
+type Backend struct {
+	Address        string
+	Port           uint
+	OverrideHost   string
+	SSLSNIHostname string
```

Comment:
NIT: I prefer to order struct fields:

```suggestion
	Address        string
	OverrideHost   string
	Port           uint
	SSLSNIHostname string
```

Reply:
Done.